scripts/style_api_gen.py

```diff
@@ -104,9 +104,7 @@ def obj_style_get(p):
 
 def get_func_cast(style):
   func_cast = ""
-  if style == 'func':
```

Comment:
@kisvegabor Are we ever going to want a function callback in the styles? If so, maybe we should keep this condition.

Reply:
It's impossible now, because func field is removed from lv_style_value_t. The generated code with 'func' can't pass the compilation.

Reply:
Oops, missed that. Sounds good.